assets/js/theme/custom/mega-menu.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Open mega menu on search button click
+ */
+export default function megaMenu() {
+    const MOBILE_WIDTH = 768
```

Comment:
Is this value used anywhere else? Maybe it would be good to keep javascript breakpoint values in one place for the sake of consistency?